wafer/talks/models.py

```diff
@@ -221,6 +222,11 @@ def has_url(self):
 
     has_url.boolean = True
 
+    @property
+    def review_score(self):
+        reviews = [review.total_score for review in self.reviews.all()]
+        return sum(reviews) / len(reviews)
+
```

Comment:
Could we add sorting on review score? I can see that being quite useful in the admin interface to look at cases that need attention.

Reply:
I think that would require putting the review score into a field on the review model. Just including it in `sortable_by` doesn't seem to do anything.

Reply:
On further reflection, what I think we need filters on review score and review count., rather than sorting. As a talk coordinator, I'd want to be able answer questions like "which talks have no reviews yet" or "which talks have at least X reviews and a score above / below Y"  from the admin view and filtering the list will enable that.